src/Forms/GridField/GridFieldPrintButton.php

```diff
@@ -126,21 +126,26 @@ public function getURLHandlers($gridField)
      */
     public function handlePrint($gridField, $request = null)
     {
+        $orig = ini_get('max_execution_time');
         set_time_limit(60);
         Requirements::clear();
 
         $data = $this->generatePrintData($gridField);
 
         $this->extend('updatePrintData', $data);
 
+        $ret = null;
         if ($data) {
-            return $data->renderWith([
+            $ret = $data->renderWith([
                 get_class($gridField) . '_print',
                 GridField::class . '_print',
             ]);
         }
+        // Reset the time limit, this is done mainly to prevent the 60 second time limit from
+        // making unrelated unit-tests run out of execution time when running a testsuite.
```

Comment:
```suggestion
        // Don't force the rest of execution to obey the print time limit
```
It's not just tests - anything after this shouldn't be restricted to this limit.